models/models.js

```diff
@@ -1,15 +1,32 @@
 const db = require('../db/connection')
 
-const fetchAllTopics = () => {
-    console.log("in model")
+const fetchAllTopics = (sort_by='slug', order='ASC', slug) => {
 
-    let queryString = 'SELECT * FROM topics'
+    const queryValues = []
+    
+    let queryString = `SELECT * FROM topics`
 
-    return db.query(queryString)
+    if (slug){
```

Comment:
try to make sure the code on each branch is relevant to the ticket you are working on. We don't yet need all of this code yet